src/main/java/uk/gov/pay/ledger/queue/eventprocessor/DisputeEventProcessor.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.pay.ledger.queue.eventprocessor;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.pay.ledger.event.model.Event;
+
+public class DisputeEventProcessor extends EventProcessor{
```

Comment:
We don't strictly need this yet. We're not going to be doing any processing of these kind of events until the second half of the chargebacks work, so this will just sit around logging. No harm in adding it though.

Reply:
I'm aware, I just thought this looks better than some // TODO 😃